bokeh/client/session.py

```diff
@@ -194,7 +194,8 @@ def show_session(session_id=None, url='default', session=None, browser=None, new
             browser (str, optional) : browser to show with (default: None)
                 For systems that support it, the **browser** argument allows
                 specifying which browser to display in, e.g. "safari", "firefox",
-                "opera", "windows-default" (see the ``webbrowser`` module
+                "opera", "windows-default" (see the
+                :doc:`webbrowser <python:library/webbrowser>` module
```

Comment:
Just speaking personally, if the choice is between a slightly longer (than 80chars) line, or a weird extra-short line, I find the longer line more readable.